### ISSUE TITLE
[BUG] Adding a playlist while a song is playing returns an error

### ISSUE THREAD

**Issue by ThaddeusKKR** [SushiBtw/discord-music-player#109]
**Bug Description**
When adding a playlist to the queue when a song is currently playing, the bot returns this error with no references:
```js
Cannot read property 'guild' of undefined
```

**Reproduction Steps**
1. Play a song
2. Add a playlist (I tested both [YouTube](https://www.youtube.com/playlist?list=PL4o29bINVT4EG_y-k5jGoOu3-Am8Nvi10) and [Spotify](https://open.spotify.com/playlist/1ubkZf1fbarE0s2jZcmpjE?si=8ea3a3215b4c47dc) playlists)
3. Observe the result.

**Expected Result**
The playlist is successfully added to the queue.

**Attachments**
`play` command code:
```js
const query = args.join(" ")
        const spotifySongRegex = (/https?:\/\/(?:embed\.|open\.)(?:spotify\.com\/)(?:track\/|\?uri=spotify:track:)((\w|-){22})/)
        const spotifyPlaylistRegex = (/https?:\/\/(?:embed\.|open\.)(?:spotify\.com\/)(?:playlist\/|\?uri=spotify:playlist:)((\w|-){22})/)
        const spotifyAlbumRegex = (/https?:\/\/(?:embed\.|open\.)(?:spotify\.com\/)(?:album\/|\?uri=spotify:album:)((\w|-){22})/)
        if (ytsr.validate(query, "PLAYLIST_ID") == true) {
            client.player.playlist(message, {
                search: args.join(' '),
                maxSongs: -1,
                requestedBy: message.author
            })
            return
        }
        if (spotifyPlaylistRegex.test(query) == true) {
            client.player.playlist(message, {
                search: args.join(' '),
                maxSongs: -1,
                requestedBy: message.author
            })
            return
        }
        if (client.player.isPlaying(message)) {
            client.player.addToQueue(message, {
                search: query,
                requestedBy: message.author
            })
            return
        }
        client.player.play(message, {
            search: query,
            requestedBy: message.author
        })
```

**Versioning:**
 - OS: Ubuntu 20.04 & Windows 10
 - NPM Version: 7.9.0
- Module Version: master (Also tested with latest)

**Additional Comments**
None

**Comment by ThaddeusKKR** [comment]
After further testing, I would like to add that the playlist is successfully added before the error event triggers.

**Comment by ZinMain** [comment]
i try edit the player.js with the help of google my edits worked if you to edit the player.js ur self here is the code https://sourceb.in/epya6uCgNA i dunno if the codes good or not but hey atleast it worked 😀

**Comment by SushiBtw** [comment]
Hey, I looked into the code and it's looking good, I will be pushing it to the npm today! Thanks ;)

**Comment by SushiBtw** [comment]
**I fixed your issue and I will be pushing the update to NPM in ~4 hours.**
I will let you know, thanks for reporting!

**Comment by SushiBtw** [comment]
Also dupe of #98

**Comment by SushiBtw** [comment]
Hey! Update time!
I hope everything got fixed, please check within `v7.1.0` and let me know if the error still exists!
https://www.npmjs.com/package/discord-music-player

Thanks for reporting,
Sushi!
